src/Repository/IncentiveImportRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
As [discussed previously](https://github.com/all-of-us/healthpro/pull/1111/files#r869516860), if we don't have any custom repository methods, let's remove the empty repository classes (and update the entity definition to remove the `repositoryClass` annotation attribute)